pkg/process/checks/container.go

```diff
@@ -41,7 +40,7 @@ type ContainerCheck struct {
 }
 
 // Init initializes a ContainerCheck instance.
-func (c *ContainerCheck) Init(cfg *config.AgentConfig, info *HostInfo) error {
+func (c *ContainerCheck) Init(_ *SysProbeConfig, info *HostInfo) error {
```

Comment:
Ideally I'd like to try to get rid of this parameter, but components will make this much cleaner, so not super worried about it 👍🏻

Reply:
yes, agreed, though with components it will become something like `sysconfig.Component` here - based on https://github.com/DataDog/datadog-agent/pull/14024